src/pybind/mgr/cephadm/templates/services/prometheus/prometheus.yml.j2

```diff
@@ -103,7 +133,26 @@ scrape_configs:
         ca_file: root_cert.pem
 {% else %}
     honor_labels: true
+    relabel_configs:
+    - source_labels: [__address__]
```

Comment:
`__address__`  is a built-in label in Prometheus that provides information about the source address of the scraped metrics, and it can be utilized in configurations for dynamic labeling or relabeling operations.
In this case `__address__` label will be replaced with `cluster` label which contains the cluster fsid.